src/index.html

```diff
@@ -10,7 +10,35 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <div class="search-bar">
+      <form
+        class="search-bar__form search-bar__form--big"
+        action="search"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          name="search"
+          type="text"
+          class="search-bar__input search-bar__input--big"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </form>
+
+      <form
+        class="search-bar__form search-bar__form--small"
+        action="search"
+        method="GET"
+        data-qa="small"
+      >
+        <input
+          name="search"
+          type="text"
+          class="search-bar__input search-bar__input--small"
+          placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
          class="search-bar__input search-bar__input--small"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
closing bracket should be on the same level as opening one